packages/devtools-client-adapters/src/types.js

```diff
@@ -43,7 +43,8 @@ export type ActorId = string;
 export type Location = {
   sourceId: SourceId,
   line: number,
-  column?: number
+  column?: number,
+  source?: any
```

Comment:
Hi @jasonLaster 

`source` is actually `Map` with `ArrayMapNode`. I'm not sure if it's possible to create this type, so I use `any`. Please let me know if we want to do this differently. Thank you! :)

<img width="573" alt="screen shot 2017-02-16 at 9 31 57 am" src="https://cloud.githubusercontent.com/assets/4562118/23006197/61fd5e34-f43b-11e6-8e0b-ff782716131b.png">